apps/devrel-fyi/src/pages/index.tsx

```diff
@@ -69,9 +69,14 @@ const Home: NextPage<{
       productId: defaultProduct?.productId,
     })
 
+  // TODO: should we move the coupon call down into `<Pricing>` so that it
+  // can be called separate for each product?
+  const productId = products[0]?.productId
+
   const {redeemableCoupon, RedeemDialogForCoupon, validCoupon} = useCoupon(
     commerceProps?.couponFromCode,
     {
+      id: productId,
```

Comment:
I put in this temporary bandaid fix for epic-react, devrel-fyi, and pro-aws since those are all in pre-purchase mode. But as the comment says, might make sense to think about moving the useCoupon call somewhere within `products.map` so that the Redeem Purchase modal will work independently for each.